validation.php

```diff
@@ -0,0 +1,25 @@
+<?php
+$db = new PDO('mysql:host=db; dbname=pokemon-cards', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT * FROM `pokemon-card`;');
+
+$query->execute();
+
+$allPokemon = $query->fetchAll();
```

Comment:
dont need to pull data into this page as none is used

Reply:
Done